controller/volume_controller_test.go

```diff
@@ -849,7 +860,7 @@ func (s *TestSuite) TestVolumeLifeCycle(c *C) {
 
 	// Salvage Requested
 	tc = generateVolumeTestCaseTemplate()
-	tc.volume.Spec.NodeID = TestNode1
+	tc.volume.Spec.NodeID = ""
```

Comment:
One question, which controller/part is responsible for unsettling this after `v.Status.Robustness = longhorn.VolumeRobustnessFaulted`?

Reply:
The VA controller https://github.com/longhorn/longhorn-manager/blob/a5b67e5ec57ec38be0d50cdbe77428d924f1b2fb/controller/volume_attachment_controller.go#L461

Reply:
NIT: Then you can add a comment that explains this is at the mid of the salvage flow. (volume is Faulted and the VA controller already unset spec.NodeID for this volume.) Or if possible, you can add one more case that explains how the volume becomes Faulted.

Reply:
I added the comment. Thank you!